_redirects

```diff
@@ -1,2 +1,3 @@
 https://a11ysd.com/* https://www.a11ysd.com/:splat 301!
 a11ysd.com/* https://www.a11ysd.com/:splat 301!
+/* /index.html 200
```

Comment:
Your redirects are not working because this file isn't included in your `dist` directory.

I suggest doing something like this:

```json
{
  "scripts": {
    "build": "parcel build src/index.html",
    "postbuild": "cp _redirects dist/"
  }
}
```

Reply:
🤦‍♂️ thanks man. You'd think I would've caught this since I've dealt with this before (https://github.com/schne324/portfolio/blob/master/webpack.config.js#L15)